larq_zoo/literature/meliusnet.py

```diff
@@ -0,0 +1,240 @@
+from typing import Optional, Sequence, Union
+
+import larq as lq
+import tensorflow as tf
+from zookeeper import Field, factory
+
+from core import utils
+from core.model_factory import ModelFactory
+
+
+@factory
+class MeliusNetFactory(ModelFactory):
+    weights: Optional[str] = Field(None)
+
+    # overall architecture configuration
+    num_blocks: Sequence[int]
+    transition_features: Sequence[int]
+    name: str
+
+    # Some default layer arguments.
+    use_conv_bias: bool = Field(False)
+    batch_norm_momentum: float = Field(0.9)
+    kernel_initializer: Optional[Union[str, tf.keras.initializers.Initializer]] = Field(
+        "glorot_normal"
+    )
+    input_quantizer = Field("ste_sign")
+    kernel_quantizer = Field("ste_sign")
+    kernel_constraint = Field("weight_clip")
```

Comment:
If I recall correctly MeliusNet uses a clipping value of `1.3` similar to `BinaryDenseNet`.